namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-test/resources/rds.tf

```diff
@@ -0,0 +1,59 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "laa_crime_apps_team_rds" {
+  source               = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.0"
```

Comment:
The RDS module is now on version 5.1 - please update this reference and check that you've set all variables required (by comparing with the example in the RDS module repo)

Reply:
Thanks. I've pushed a commit to change the module to 5.1. The variables look correct to me in comparison to the example module, but let me know if I've missed anything.